ivc/src/ivc/mod.rs

```diff
@@ -16,6 +19,14 @@ pub const N_LIMBS_XLARGE: usize = 2;
 // [folding::tests::test_quadraticization]
 pub const N_ADDITIONAL_WIT_COL_QUAD: usize = 48;
 
+/// Number of challenges in the IVC circuit.
+/// It is the maximum number of constraints per row.
+/// We do suppose the Poseidon circuit has the highest number of constraints,
+/// for now.
+/// We also add the number of "blocks" in the IVC circuit as there will be
+/// [N_BLOCKS] alphas required to aggregate all blocks on each row.
+pub const IVC_NB_CHALLENGES: usize = IVC_POSEIDON_NB_CONSTRAINTS + N_BLOCKS;
```

Comment:
Right now this will not be enough: folding needs the /sum/ of constraints, not the max(block_constraints) + common_constraints. To be optimised in the future.